releasenotes/notes/aggregates-api-output-change-2bc6620c7f595925.yaml

```diff
@@ -0,0 +1,6 @@
+---
+fixes:
+  - |
+    Aggregates API output introduced in 4.1.0 doesn't permit to easly identify which timeseries
```

Comment:
s/permit to easly identify which timeseries/allow for easy identification of which timeseries/